index.html

```diff
@@ -64,8 +64,31 @@
     <h2>
       Introduction
     </h2>
+    <p>
+      This document defines an API enabling a website to request presentation of a verifiable digital credential (VDC).
```

Comment:
Automagically adds the (VDC) and auto titles it globally: 
 
```suggestion
      This document defines an API enabling a website to request presentation of a <dfn data-abbr>Verifiable Digital Credential</dfn>.
```